pages/dao/[symbol]/proposal/new.tsx

```diff
@@ -0,0 +1,260 @@
+  const [form, setForm] = useState({
+    title: '',
+    description: '',
+  })
+  const [formErrors, setFormErrors] = useState({})
+  const [instructionsComponents, setInstructions] = useState([
+    { ...defaultInstructionComponentModel, type: availabileInstructions[0] },
+  ])
+
+  const handleSetForm = ({ propertyName, value }) => {
+    setFormErrors({})
+    setForm({ ...form, [propertyName]: value })
+  }
+  const setInstructionType = ({ value, idx }) => {
+    const newInstruction = {
+      type: value,
+    }
+    setInstructionChangeAtIndex({ newModel: newInstruction, idx })
+  }
+  const setInstructionChangeAtIndex = ({ newModel, idx }) => {
+    const newInstructions = [...instructionsComponents]
+    newInstructions[idx] = newModel
+    setInstructions(newInstructions)
+  }
+  const addInstruction = () => {
+    setInstructions([
+      ...instructionsComponents,
+      defaultInstructionComponentModel,
+    ])
+  }
+  const removeInstruction = (idx) => {
+    setInstructions([
+      ...instructionsComponents.filter((x, index) => index !== idx),
+    ])
+  }
+  const getInstructions = async () => {
+    const instructions: Instruction[] = []
+    const instRefs: SplTokenTransferRef[] = refs.current
+    for (const inst of instRefs) {
+      const instruction: Instruction = await inst?.getSerializedInstruction()
+      instructions.push(instruction)
+    }
+    return instructions
+  }
+  const handleCreate = async (isDraft) => {
+    setFormErrors({})
+
+    const { isValid, validationErrors }: formValidation = await isFormValid(
+      schema,
+      form
+    )
+    const instructions: Instruction[] = await getInstructions()
+    let proposalAddress: PublicKey | null = null
+    if (!realm) {
+      throw 'no realm selected'
+    }
+    if (isValid && instructions.every((x: Instruction) => x.isValid)) {
+      const sourceAccount = instructions[0].sourceAccount
+      if (!sourceAccount) {
+        throw 'no source account selected'
+      }
+      if (!ownTokenRecord) {
+        throw 'no own token record found'
+      }
+      const rpcContext = new RpcContext(
+        new PublicKey(realm.account.owner.toString()),
+        wallet,
+        connection.current,
+        connection.endpoint
+      )
+      const instructionsData = instructions.map((x) =>
+        getInstructionDataFromBase64(x.serializedInstruction)
+      )
+      const params: createParams = [
+        rpcContext,
+        realm.pubkey,
+        sourceAccount.pubkey,
+        ownTokenRecord?.pubkey,
+        form.title,
+        form.description,
+        realm.info.communityMint,
+        sourceAccount?.info?.config.minInstructionHoldUpTime,
+        sourceAccount?.info?.proposalCount,
+        instructionsData,
+      ]
+      try {
+        proposalAddress = isDraft
+          ? await createProposalDraft(...params)
+          : await createProposal(...params)
+        notify({
+          message: isDraft
```

Comment:
This seem to be unnecessary because `createProposalDraft` and `createProposal` already show success message